docs/croissant-spec-draft.md

```diff
@@ -917,21 +917,21 @@ A `Field` is part of a `RecordSet`. It may represent a column of a table, or a n
   </tr>
   <tr>
     <td>references</td>
-    <td>Reference</td>
+    <td>Field <code>@id</code></td>
     <td>MANY</td>
-    <td>Another <code>Field</code> of another <code>RecordSet</code> that this field references. This is the equivalent of a foreign key reference in a relational database.</td>
+    <td>An unordered list of references to other <code>RecordSet</code> <code>Field</code>s. This is the equivalent of a foreign key reference in a relational database.</td>
   </tr>
   <tr>
     <td>subField</td>
-    <td>Field</td>
+    <td>Field <code>@id</code></td>
     <td>MANY</td>
-    <td>Another <code>Field</code> that is nested inside this one.</td>
+    <td>An unordered list of references to <code>Field</code>s that are nested within this one.</td>
   </tr>
   <tr>
     <td>parentField</td>
-    <td>Reference</td>
+    <td>Field <code>@id</code></td>
     <td>MANY</td>
-    <td>A special case of <code>SubField</code> that should be hidden because it references a <code>Field</code> that already appears in the <code>RecordSet</code>.</td>
+    <td>An unordered list of references to <code>Field</code>s. A special case of <code>SubField</code> that should be hidden because it references a <code>Field</code> that already appears in the <code>RecordSet</code>.</td>
```

Comment:
Many of the properties that reference other are marked as "MANY".
Should property names be pluralized?
Should they accept "ONE or MANY"?